micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jetty/JettyStatisticsMetrics.java

```diff
@@ -0,0 +1,107 @@
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.jetty;
+
+import io.micrometer.core.instrument.FunctionCounter;
+import io.micrometer.core.instrument.FunctionTimer;
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.TimeGauge;
+import io.micrometer.core.instrument.binder.MeterBinder;
+
+
+import org.eclipse.jetty.server.handler.StatisticsHandler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.function.ToDoubleFunction;
+import java.util.function.ToLongFunction;
+
+public class JettyStatisticsMetrics implements MeterBinder {
+    private final StatisticsHandler statisticsHandler;
+    private static final List<Tag> EMPTY = new ArrayList<>();
+
+    public JettyStatisticsMetrics(StatisticsHandler statisticsHandler) {
+        this.statisticsHandler = statisticsHandler;
+    }
+
+    @Override
+    public void bindTo(MeterRegistry reg) {
+        bindTimer(reg,"jetty.requests", "Request duration", StatisticsHandler::getRequests, StatisticsHandler::getRequestTimeTotal);
+        bindTimer(reg,"jetty.dispatched", "Dispatch duration", StatisticsHandler::getDispatched, StatisticsHandler::getDispatchedTimeTotal);
+
+        bindCounter(reg,"jetty.async.requests", "Total number of async requests", StatisticsHandler::getAsyncRequests);
+        bindCounter(reg,"jetty.async.dispatches", "Total number of requested that have been asynchronously dispatched", StatisticsHandler::getAsyncDispatches);
+        bindCounter(reg,"jetty.async.expires", "Total number of async requests requests that have expired", StatisticsHandler::getExpires);
+        bindCounter(reg,"jetty.responses.bytes", "Total number of bytes across all responses", StatisticsHandler::getResponsesBytesTotal);
+
+        bindGauge(reg,"jetty.requests.active", "Number of requests currently active", StatisticsHandler::getRequestsActive);
+        bindGauge(reg,"jetty.dispatched.active", "Number of dispatches currently active", StatisticsHandler::getDispatchedActive);
+        bindGauge(reg,"jetty.dispatched.active.max", "Maximum number of active dispatches being handled", StatisticsHandler::getDispatchedActiveMax);
+        bindGauge(reg,"jetty.dispatched.time.max", "Maximum time spent in dispatch handling", StatisticsHandler::getDispatchedTimeMax);
+        bindGauge(reg,"jetty.async.requests.waiting", "Currently waiting async requests", StatisticsHandler::getAsyncRequestsWaiting);
+        bindGauge(reg,"jetty.async.requests.waiting.max", "Maximum number of waiting async requests", StatisticsHandler::getAsyncRequestsWaitingMax);
+
+        bindTimeGauge(reg,"jetty.request.time.max.seconds", "Maximum time spent handling requests", StatisticsHandler::getRequestTimeMax);
```

Comment:
"seconds" is inappropriate, depends on the base time unit of the backend. This is appended automatically by the Prometheus naming convention

Reply:
Removed the seconds, good catch